builder/proxmox/common/step_start_vm.go

```diff
@@ -334,6 +355,13 @@ func generateProxmoxEfi(efi efiConfig) proxmox.QemuDevice {
 	return dev
 }
 
+func formatBoolPointer(v *bool) string {
+	if v == nil {
+		return ""
+	}
+	return strconv.FormatBool(*v)
+}
```

Comment:
Didn't see any examples of optional values in the project, so came up with this. The use case is for setting a default bool value if none was given. Let me know if there is a better way to handle "unset" values 👍

Reply:
Thanks, these are excellent suggestions! I really like the idea of just inverting the flag 💯